src/components/TextInput/component.tsx

```diff
@@ -83,6 +90,15 @@ export const Component = ({
       {!!description && (
         <Description data-testid={buildTestId('description')}>{description}</Description>
       )}
+      {!!validationMessages && (
```

Comment:
Maybe check here that `validationMessages` contains at least one item, so that we can avoid rendering an empty message container.